students/darrell_h/lesson_4/mailroom_enhanced.py

```diff
@@ -36,6 +39,23 @@ def add_donation(donor_name, exsting_donor):
             d[donor_name] = [int(donation)]
             break
 
+def send_to_screen():
+    for k, v in d.items():
```

Comment:
You should avoid single letter variable names.  Spell out what the variable is or others (or you!) might not understand what it is when looking at your code.